GameCode

```diff
@@ -47,7 +50,7 @@ const determineWinner = (userChoice, computerChoice) => {
 };
 
 const playGame = () => {
-    const userChoice = getUserChoice('scissors');
```

Comment:
I still don't understand how this input would be pulled if this was ostensibly to be played as a legit game and not just some ramshackle learnin' code but I guess that's a functionality that I'll learn later? Not knowing things makes me anxious.